src/Homework3/Book.java

```diff
@@ -0,0 +1,54 @@
+package Homework3;
+
+import java.util.*;
```

Comment:
Wildcard imports are bad :)

Reply:
Changed it to **import java.util.List**
And also in src/Homework3/Task.java to 
**import java.util.Arrays;
import java.util.List;**